src/site/stages/build/drupal/graphql/pressReleasePage.graphql.js

```diff
@@ -79,7 +79,7 @@ const GetNodePressReleasePages = `
 
   ${pressReleasePage}
 
-  query GetNodeVaForms($onlyPublishedContent: Boolean!) {
+  query GetNodePressRelease($onlyPublishedContent: Boolean!) {
```

Comment:
This would only have been visible in error output. The export name `GetNodePressReleasePages` is what was important.